row_length: 6.0
row_seperation_dist: 0.7
num_rows: 4
begin_from_left_row: True
